package/yast2-bootloader.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jan 28 22:37:00 UTC 2022 - Josef Reidinger <jreidinger@suse.com>
+
+- Allow product/system role to select boot timeout (jsc#SLE-22667)
+- 4.4.16
```

Comment:
Just miss the version bump in the spec file.

Reply:
good catch